From c3fe36de35403cdafc7c8c8506237281df069567 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Sun, 12 Aug 2018 19:06:12 -0700
Subject: [PATCH 1/2] p7zip: Cleanup if statement braces, whitespace lines, and
 ifs without paranthesis)

Change-Id: I7edff09a0ef5bd262187f707fa47a97da4d5a33e

# Conflicts:
#	CPP/7zip/Archive/Chm/ChmHandler.cpp
#	CPP/7zip/Archive/ComHandler.cpp
---
 CPP/7zip/Archive/7z/7zIn.cpp        |  4 ++--
 CPP/7zip/Archive/ArHandler.cpp      |  6 ++----
 CPP/7zip/Archive/Chm/ChmHandler.cpp | 23 +++++++++++++----------
 CPP/7zip/Archive/ComHandler.cpp     |  4 ++--
 4 files changed, 19 insertions(+), 18 deletions(-)

diff --git a/CPP/7zip/Archive/7z/7zIn.cpp b/CPP/7zip/Archive/7z/7zIn.cpp
index 7c6dde2..4cf567c 100644
--- a/CPP/7zip/Archive/7z/7zIn.cpp
+++ b/CPP/7zip/Archive/7z/7zIn.cpp
@@ -1510,8 +1510,8 @@ HRESULT CInArchive::ReadDatabase2(
     unsigned i;
     for (i = checkSize - 2;; i--)
     {
-      if (buf[i] == NID::kEncodedHeader && buf[i + 1] == NID::kPackInfo ||
-          buf[i] == NID::kHeader && buf[i + 1] == NID::kMainStreamsInfo)
+      if ((buf[i] == NID::kEncodedHeader && buf[i + 1] == NID::kPackInfo) ||
+          (buf[i] == NID::kHeader && buf[i + 1] == NID::kMainStreamsInfo))
         break;
       if (i == 0)
         return S_FALSE;
diff --git a/CPP/7zip/Archive/ArHandler.cpp b/CPP/7zip/Archive/ArHandler.cpp
index 83ed5a0..01f384a 100644
--- a/CPP/7zip/Archive/ArHandler.cpp
+++ b/CPP/7zip/Archive/ArHandler.cpp
@@ -618,11 +618,9 @@ STDMETHODIMP CHandler::Open(IInStream *stream,
       _items.DeleteFrontal(1);
       for (unsigned i = 0; i < _items.Size(); i++)
         if (_items[i].Name.IsPrefixedBy("data.tar."))
-          if (_mainSubfile < 0)
+          if (_mainSubfile < 0) {
             _mainSubfile = i;
-          else
-          {
-            _mainSubfile = -1;
+          } else {
             break;
           }
     }
diff --git a/CPP/7zip/Archive/Chm/ChmHandler.cpp b/CPP/7zip/Archive/Chm/ChmHandler.cpp
index b890d8f..39c78c4 100644
--- a/CPP/7zip/Archive/Chm/ChmHandler.cpp
+++ b/CPP/7zip/Archive/Chm/ChmHandler.cpp
@@ -23,6 +23,7 @@
 
 using namespace NWindows;
 using namespace NTime;
+using namespace NArchive::NChm;
 
 namespace NArchive {
 namespace NChm {
@@ -143,29 +144,31 @@ STDMETHODIMP CHandler::GetProperty(UInt32 index, PROPID propID, PROPVARIANT *val
     case kpidIsDir:  prop = item.IsDir(); break;
     case kpidSize:  prop = item.Size; break;
     case kpidMethod:
-    {
-      if (!item.IsDir())
-        if (item.Section == 0)
+      if (!item.IsDir()) {
+        if (item.Section == 0) {
           prop = "Copy";
-        else if (item.Section < m_Database.Sections.Size())
+        } else if (item.Section < m_Database.Sections.Size()) {
           prop = m_Database.Sections[(unsigned)item.Section].GetMethodName();
+        }
+      }
       break;
-    }
     case kpidBlock:
-      if (m_Database.LowLevel)
+      if (m_Database.LowLevel) {
         prop = item.Section;
-      else if (item.Section != 0 && item.Section < m_Database.Sections.Size())
+      }
+      else if (item.Section != 0 && item.Section < m_Database.Sections.Size()){
         prop = m_Database.GetFolder(index);
+      }
       break;
-    
+
     #ifdef _CHM_DETAILS
-    
+
     case kpidSection:  prop = (UInt32)item.Section; break;
     case kpidOffset:  prop = (UInt32)item.Offset; break;
 
     #endif
   }
-  
+
   prop.Detach(value);
   return S_OK;
   COM_TRY_END
diff --git a/CPP/7zip/Archive/ComHandler.cpp b/CPP/7zip/Archive/ComHandler.cpp
index c6d2bd2..64f8df5 100644
--- a/CPP/7zip/Archive/ComHandler.cpp
+++ b/CPP/7zip/Archive/ComHandler.cpp
@@ -583,8 +583,8 @@ HRESULT CDatabase::Open(IInStream *inStream)
     {
       // bool isThereExt = (msiName.Find(L'.') >= 0);
       bool isMsiSpec = (msiName[0] == k_Msi_SpecChar);
-      if (msiName.Len() >= 4 && StringsAreEqualNoCase_Ascii(msiName.RightPtr(4), ".cab")
-          || !isMsiSpec && msiName.Len() >= 3 && StringsAreEqualNoCase_Ascii(msiName.RightPtr(3), "exe")
+      if ((msiName.Len() >= 4 && StringsAreEqualNoCase_Ascii(msiName.RightPtr(4), ".cab"))
+          || (!isMsiSpec && msiName.Len() >= 3 && StringsAreEqualNoCase_Ascii(msiName.RightPtr(3), "exe"))
           // || !isMsiSpec && !isThereExt
           )
 
-- 
2.17.1

